guides/common/modules/proc_recovering-corrupted-content-on-a-smart-proxy.adoc

```diff
@@ -0,0 +1,32 @@
+[id="Recovering_Corrupted_Content_on_{smart-proxy-context}_{context}"]
+= Recovering corrupted content on {SmartProxy}
+
+In case of content corruption on a {SmartProxy}, you can recover it by using the `verify-checksum` command in Hammer CLI.
+The `verify-checksum` command repairs content filtered for content views, lifecycle environments, repositories or all content without any filters on the {SmartProxy}.
+After running a `verify-checksum` command, it will appear on the *Tasks* page as the action *Verify checksum for content on smart proxy*.
+
+.CLI procedure
+* To repair content in a content view, run Hammer on your {SmartProxy}:
++
+[options="nowrap", subs="+quotes,attributes"]
+----
+$ {hammer-smart-proxy} content verify-checksum --id 2 --organization-id 1 --content-view-id 3
+----
+* To repair content in a lifecycle environment, run Hammer on your {SmartProxy}:
++
+[options="nowrap", subs="+quotes,attributes"]
+----
+$ {hammer-smart-proxy} content verify-checksum --id 2 --organization-id 1 --lifecycle-environment-id 1
+----
+* To repair content in a repository, run Hammer on your {SmartProxy}:
++
+[options="nowrap", subs="+quotes,attributes"]
+----
+$ {hammer-smart-proxy} content verify-checksum --id 2 --organization-id 1 --repository-id 1
+----
```

Comment:
For these commands, perhaps clarify what the ID is of, too?
```
... verify-checksum --id __My_{smart-proxy-context-titlecase}_ID__ ...
```

Reply:
@sjha4 Perhaps you can help clarify here?

Reply:
The capsule content verify-checksum requires the id of the smart proxy to run the checksum verification and repair on.
The base command is :
{hammer-smart-proxy} content verify-checksum --id 2  where `id` is the smart proxy id.
The filters available to further narrow down the task to  target specific content are:
a)  --content-view-id
b) --lifecycle-environment-id
c) --repository-id

Reply:
@bangelic What is your reason not to incorporate this information? Do you find the commands clear enough? As in that it's inambiguous what the `--id` is referring to?

Reply:
@Lennonka I thought the last sentence I added `The ID of the {SmartProxy} is required to run the `verify-checksum` command.`explained it. Should I reword it?